### PR TITLE
CMS-312 – Admin Change Header/Toolbar

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- https://divio-ch.atlassian.net/browse/CMS-312
- Please ignore compiled css file
- and please ignore settings file as well
- will assign this ticket to Matthias that he check button and logout/password links positions

Original Toolbar from Header:

screen shot 2015-09-01 at 15 24 03

Toolbar on Admin:

screen shot 2015-09-01 at 15 29 30

### CODE REVIEW

File: djangocms_admin_style/sass/djangocms-admin.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
-@import "_settings.scss";
+@import "settings/_settings.scss";
```
Comment (vxsx) [inline djangocms_admin_style/sass/djangocms-admin.scss]: we should probably name this one `_cms.scss` and have `_all.scss` in settings folder so we can reference them from there

**Comment by vxsx** [comment]
looks nice. is the `back to ...` button always on the left now?
but anyway overall :+1:

**Comment by lory87** [comment]
@vxsx updated settings structure. As I wrote in the ticket notes I will create a ticket for Matthias that he can check the position of the links and the button. :)

**Comment by vxsx** [comment]
perfect :boom:

**Pull request merged**